### PR TITLE
Fan-in NSM Releasing

### PR DESCRIPTION
**Pull request by NikitaSkrynnik**
## Issue link
https://github.com/networkservicemesh/deployments-k8s/issues/4271

### CODE REVIEW

File: .github/workflows/pr-for-release.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+name: Pull Request on release/* Branch Push
+on:
+  push:
+    branches:
+      - "release/**"
+      - "!release/*"
```
Comment (denis-tingaikin) [inline .github/workflows/pr-for-release.yaml:7]: Could you explain this line?

Comment (NikitaSkrynnik) [reply .github/workflows/pr-for-release.yaml:7]: This line means that we want to create PR for any branch `release/networkservicemesh/*/v*.*.*`, but don't want to create PR for branch `release/v*.*.*`
`**` - include any character
`*` - include any character except `\`

Comment (denis-tingaikin) [reply .github/workflows/pr-for-release.yaml:7]: Got it.
This sounds like a delicate moment that I guess should be pointed by comment for this line in the file.

**Review by denis-tingaikin — APPROVED** [review]
Looks good, only one cosmetic comment to fix.

@glazychev-art 

Could you also have a look?

File: .github/workflows/pr-for-release.yaml (hunk 1 above)
Comment (NikitaSkrynnik) [reply .github/workflows/pr-for-release.yaml:7]: ok

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**